### PR TITLE
[WIP] How rich am I calculator component

### PR DESCRIPTION
**Pull request by fellmirr**
Adds a "how rich am I" calculator component, based on the GWWC version but adapted to Norwegian context. 

Remaining tasks:
- [x] Add number of adults and children in household #612 
- [x] Add impact estimate section #613 
- [x] Document data sources and key uncertainties
- [ ] Document the components thoroughly for easier reuse by other orgs

---

Tested on devices 
- [x] Desktop 💻
- [x] Mobile 📱

Tests  
- [x] All tests are running ✔️
- [ ] Test are updated 🧪
- [x] Code Review 👩‍💻
- [x] QA 👌

⏲️ Time spent on CR: < 2 hr

⏲️ Time spent on QA: > 20hr

### CODE REVIEW

File: pages/min-side/[[...slug]].tsx
Comment (EliseAndresen) [inline pages/min-side/[[...slug]].tsx:1]: Console log :P

File: pages/min-side/[[...slug]].tsx [hunk 1]
```diff
@@ -106,7 +106,6 @@ const Home: LayoutPage<{ data: any }> = ({ data }) => {
       filterYear = router.query.slug[1];
     }
   }
-  console.log(filterYear);
```
Comment (EliseAndresen) [inline pages/min-side/[[...slug]].tsx:109]: Console log :P

**Review by EliseAndresen — APPROVED** [review]

**Comment by EliseAndresen** [comment]
Mobile:

- [ ] Slider and text above it should be below graph
- [ ] User should have a submit button underneath input fields that scrolls the user to next screen (graph and slider)
- [ ] View is slightly bigger than screen so it slides sideways making it harder to use the slider
- [ ] Impact buttons look squished on mobile, could they be stacked in small views? (might outside scope for this task)

Desktop:
- [ ] Increase font size on the graph annotation

General:
- [ ] Add sentence about current percentile to text under input fields
- [ ] Add bold format to % in graph annotation

File: pages/min-side/[[...slug]].tsx (hunk 1 above)
Comment (fellmirr) [reply pages/min-side/[[...slug]].tsx:109]: Yep, but the change is that I'm removing it :P

Comment (EliseAndresen) [reply pages/min-side/[[...slug]].tsx:109]: My bad 🤦‍♀️

**Pull request merged**